src/tests/lib/qunit_config.js

```diff
@@ -1 +1,41 @@
-QUnit.config.testTimeout = 6400;
+/* globals badger:false */
+
+(function () {
+
+  let BACKUP = {};
+
+  QUnit.config.testTimeout = 6400;
+
+  // runs before any tests begin
+  QUnit.begin(() => {
+    // disable storage persistence
+    chrome.storage.local.set = () => {};
+
+    // set defaults
+    var settings = badger.storage.getBadgerStorageObject("settings_map");
+    for (let key in badger.defaultSettings) {
+      settings.setItem(key, badger.defaultSettings[key]);
```

Comment:
That is a good point.

Maybe we could also override `chrome.storage.local.get` so when we initialize PB in the test page, it will start with default data. What do you think @ghostwords ?